lib/mlir/Dialect/Clift/IR/CMakeLists.txt

```diff
@@ -21,3 +21,9 @@ add_mlir_dialect_library(
   MLIRIR
   PRIVATE
   revng::revngModel)
+
```

Comment:
Please add a TODO around all the usage of `add_mlir_dialect_library` to switch to `revng_add_library`.

`revng_add_library` does things that are important and we're bypassing it.

Reply:
I've added this in the internal Clift to-do document. Is that okay, or do you also want comments in the CMake files?

Reply:
Add a comment in the CMake file too. Comments show that we now that the code we're looking at has a known limitation.